### PR TITLE
Add support for github actions

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**

### CODE REVIEW

**Comment by yann300** [comment]
might need a little rebasing

File: .github/actions/directory/src/index.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+"use strict";
```
Comment (yann300) [inline .github/actions/directory/src/index.js]: do you need this `index.js` file? that's the one executed by github job?

File: .github/actions/directory/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { setFailed, getInput, debug, warning } from '@actions/core';
+import { context, GitHub } from '@actions/github';
+import { ReposGetContentsResponseItem } from '@octokit/rest';
+
+async function createFile() {
+  try {
+    const token = getInput('GITHUB_TOKEN');
+
+    debug('Inside try block');
```
Comment (yann300) [inline .github/actions/directory/src/index.ts]: this should be removed i guess

**Comment by yann300** [comment]
with that PR, what will be the raw url of the built list of profiles? cause we then have to update https://github.com/ethereum/remix-ide/pull/2342/files#diff-8adf04c5a6537319153b09ecbbb10fa4R23

File: .github/actions/directory/src/index.js (hunk 1 above)
Comment (GrandSchtroumpf) [reply .github/actions/directory/src/index.js]: no, this is the intermediary file : 
`src/index.ts` -> `src/index.js` -> `dist/index.js`
I'll update the .gitignore

**Pull request merged**